aws/resource_aws_opsworks_custom_layer_test.go

```diff
@@ -308,7 +386,20 @@ resource "aws_opsworks_custom_layer" "tf-acc" {
     mount_point = "/home"
     size = 100
     raid_level = 0
-  }
+  },
+  autoscaling = true
```

Comment:
Could you please split this out into its own acceptance test configuration and acceptance test function? We should verify that not having these specified does not generate any unexpected differences. 👍